packages/example/src/hooks/useSnap.ts

```diff
@@ -0,0 +1,69 @@
+import {
+  isSnapInstalled,
+  isMetaMaskFlaskAvailable,
+  AleoSnap,
+} from '@chainsafe/aleo-snap-adapter';
+import { MetamaskRpcRequest } from '@chainsafe/aleo-snap-shared';
+import { useCallback, useEffect, useState } from 'react';
+
+const isDev = true;
```

Comment:
Check out https://vitejs.dev/guide/env-and-mode.html#env-variables for `import.meta.env.DEV`